tests/functional/config.js

```diff
@@ -24,7 +24,11 @@ let config = {
   sync: true,
   connectionRetryTimeout: 5000,
   connectionRetryCount: 3,
-  capabilities: [chrome],
```

Comment:
Okay, I'll revert this change, probably best to stick with what's in master already.